ladybug_comfort/wbgt.py

```diff
@@ -0,0 +1,132 @@
+# coding=utf-8
+"""Utility functions for calculating the Wet Bulb Globe Temperature (WBGT)."""
+from __future__ import division
+
+import math
+
+
+def wbgt_outdoors(ta, mrt, ws, vp):
+    """
+    The wet-bulb globe temperature (WBGT) is a type of apparent temperature,
+    and is widely used heat stress index globally (ISO 7243). It is used to
+    estimate the effect of temperature, humidity, wind speed (wind chill), and
+    visible and infrared radiation (usually sunlight) on humans.
+
+    WBGT outdoor formula is based on a mathematical model published by
+    K. Blazejczyk et al. in 2014 [1] with the natural wet bulb temparature
+    (Tnwb) based on the code written by Nick Burns [2].
+
+    Note:
+        [1] Heat stress and occupational health and safety spatial
+        and temporal differentiation, K. Blazejczyk, J.Baranowski, A. Blazejczyk,
+        Miscellanea geographica regional studies on development, Vol. 18,
+        No. 1, 2014
+        [2] https://github.com/nickb-/Calculating-WBGT/blob/master/relaxation_Tw/
+        working_code/wbgt.R
```

Comment:
I copied this hyperlink from the document in the legacy code, however, this repo is no longer accessible. This repo is where the legacy code used to calculate natural wet bulb temperature (Tnwb), which seems to be different from the formula in the paper in [1]. (see comment for line 38)